CovertInfiltration/Src/CovertInfiltration/Classes/X2EventListener_Infiltration.uc

```diff
@@ -90,6 +90,7 @@ static function CHEventListenerTemplate CreateStrategyListeners()
 	Template.AddCHEvent('LowSoldiersCovertAction', PreventLowSoldiersCovertActionNag, ELD_OnStateSubmitted, 99);
 	Template.AddCHEvent('OverrideAddChosenTacticalTagsToMission', OverrideAddChosenTacticalTagsToMission, ELD_Immediate, 99);
 	Template.AddCHEvent('PreCompleteStrategyFromTacticalTransfer', PreCompleteStrategyFromTacticalTransfer, ELD_Immediate, 99);
+	Template.AddCHEvent('HasIncreasedSquadSize', HasIncreasedSquadSize, ELD_Immediate, 99);
```

Comment:
The hook/event name is too ambiguous - e.g. one could think that it controls the number of slots on the squad select. I suggest `AllowNoSquadSizeUpgradeAchivement`.

Speaking of hooks - I don't see a CHL issue/pr?